app/views/user/manage.phtml

```diff
@@ -46,12 +46,48 @@
 		</div>
 	</form>
 
+	<form method="post" action="<?php echo _url('user', 'update'); ?>">
+		<input type="hidden" name="_csrf" value="<?php echo FreshRSS_Auth::csrfToken(); ?>" />
+		<legend><?php echo _t('admin.user.update_users'); ?></legend>
+
+		<div class="form-group">
+			<label class="group-name" for="current_user"><?php echo _t('admin.user.selected'); ?></label>
+			<div class="group-controls">
+				<select id="current_user" class="select-change" name="username">
+					<option selected="selected">&nbsp;</option>
```

Comment:
Good idea, and could you please do the same think (i.e. empty first option) for the next list of users a bit further down?
Detail: I would prefer a simple space, or Unicode non-breaking space, as I am not so found of entities.

Reply:
No pbs for the space.
On the other hand, I've tried to replicate the same thing for the other list but there is JS involved in that one. It's not that easy and I didn't take the time to dig into it.

Reply:
I've found a way :)